src/tacacs/nss/0001-Modify-user-map-profile.patch

```diff
@@ -463,7 +463,7 @@ index 79e62b9..ecfa0b0 100644
 +{
 +    char *token;
 +    char delim[] = ";\n\r";
-+    int priv = 0;
++    int priv = -1;
```

Comment:
The default value of “priv_level” in function got_tacacs_user() also need to be changed

Reply:
TACACS priv-level 0 is seldom used and defined as non-privilege in general. Any scenarios need to support it?

Reply:
does non-privilege mean normal user?

Reply:
```
privilege level 0 = seldom used, but includes 5 commands: disable, enable, exit, help, and logout
```

it is still valid, we should still enable it.

https://www.cisco.com/c/en/us/support/docs/security-vpn/remote-authentication-dial-user-service-radius/13860-PRIV.html

Reply:
Why do we need to modify default value in got_tacacs_user()? I think current behavior is that to treat user as least privileged (0) if search fails?

Reply:
The original design is that the priv-lvl for the user configuration on the TACACS + server should be set as a valid value(1-15) explicitly. If priv-lvl search fails, it will be set as privileged(0) and not allowed to login in.  
If priv-lvl(0) is treated as valid value, the default value should be set as invalid value. Unless we don't care priv-lvl.

Reply:
I think non-privilege means the user has less command privilege. There is no authorization for current SONiC cli, so the original design is to filter non-privilege user to avoid unsafe operation. Maybe it's too hard-coded to limit user's configuration.